nengo_loihi/tests/test_conv.py

```diff
@@ -592,7 +599,7 @@ def set_partition(partition=os.environ.get("PARTITION", None)):
         elif pop_type == 16:
             request.addfinalizer(set_partition)
             # multichip pop_type = 16 works only on nahuku32 board currently
-            set_partition("nahuku32")
+            set_partition("nahuku32", lmt_options="--skip-power=1")
```

Comment:
Very minor, but I'd pass both as kwargs (i.e. `partition="nahuku32", lmt_options="--skip-power=1"`).

Reply:
That seemed a bit redundant since the function was already called `set_partition`, so I renamed the function to `set_env` and passed them both as kwargs.